frontend/src/App.css

```
@@ -44,3 +44,8 @@
 .read-the-docs {
   color: #888;
 }
+
+html.dark svg,
+.svg-lightbox {
```

This always applies to `.svg-lightbox`, even in light mode, which I can hardly imagine is intended.

```suggestion
html.dark svg,
html.dark .svg-lightbox {
```